holder.go

```diff
@@ -70,7 +70,7 @@ func NewHolder() *Holder {
 }
 
 // Open initializes the root data directory for the holder.
-func (h *Holder) Open() error {
+func (h *Holder) Open(logOutput io.Writer) error {
```

Comment:
Might be better to have `NewHolder` take an argument (io.Writer), and then set its LogOutput based on that.